README.md

```diff
@@ -35,7 +35,7 @@ pip install yowsup2
 
 You need to have installed python headers (from probably python-dev package) and ncurses-dev, then run
 ```
-python setup.py install
+sudo python setup.py install
```

Comment:
"run `python setup.py install` as root" sounds better and doesn't restricts it to `sudo`